README.md

```diff
@@ -14,7 +14,17 @@ contrib | core
 ------- | ----
 master  | master
 
-You may want to take a look at the individual blocklet directories or 
-the [contribution guidelines](https://github.com/vivien/i3blocks-contrib/blob/master/CONTRIBUTING.md).
+You may want to take a look at the individual blocklet directories,
+which contain descriptions, screenshots, and installation/configuration 
+instructions.
+Note: configurations reference `$SCRIPT_DIR`, meaning the directory that you put
+the script into.
+`$SCRIPT_DIR` is not an environmental variable, you must fill it in
```

Comment:
$SCRIPT_DIR *can* be an environment variable, in fact on my system it is. I put my custom scripts in there and use `command=$SCRIPT_DIR/my_script`